website/docs/installation/weave-gitops.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Weave GitOps
+sidebar_position: 2
+hide_title: true
+pagination_next: "getting-started"
+---
+
+## Installing Weave GitOps
+
+#### Install Flux
+
+Weave GitOps is an extension to Flux and therefore requires that Flux 0.32 or later has already been installed on your Kubernetes cluster. Full documentation is available at: [https://fluxcd.io/docs/installation/](https://fluxcd.io/docs/installation/).
```

Comment:
```suggestion
```
We have a section for installing Flux on this very page so telling people they need to have Flux already installed just to tell them how to install Flux right after is a little strange.